extensions/podman/packages/extension/src/extension.spec.ts

```diff
@@ -330,7 +330,7 @@ beforeEach(() => {
   vi.resetAllMocks();
   console.error = consoleErrorMock;
 
-  (extensionApi.env.isMac as boolean) = true;
+  vi.mocked(extensionApi.env).isMac = false;
```

Comment:
by default all test had the isMac to true, but the env.isMac was false, very hard to understand the test, so making it to false is better, to force test to have to be explicit if it need to be on mac.